neoutils/neoutils.go

```diff
@@ -76,17 +78,17 @@ func EnsureIndexes(im IndexManager, indexes map[string]string) error {
 
 // EnsureConstraints will, for a map of labels and properties, check whether a constraint exists for a given property on a given label, and
 // if missing will create one. Creating the unique constraint ensures an index automatically.
-func EnsureConstraints(im IndexManager, indexes map[string]string) error {
+func EnsureConstraints(im IndexManager, indexes map[string]string, log *logger.UPPLogger) error {
```

Comment:
We can make the logger optional here as well meaning allowing to give nil as logger object.